easybuild/easyconfigs/s/Seaborn/Seaborn-0.9.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
```

Comment:
being picky ... I like to have an empty line between header and `easyblock` line to improve readability

Reply:
That's really pushing it imho, we've never required this (and if we should start doing it, we should make the tests trip over it).

Let's not try and come up with ways to keep PRs open, there's enough of them already ;)